angular-reactive/src/app/quotes.component.ts

```diff
@@ -46,7 +47,9 @@ export class QuotesComponent {
     this.selectedQuote = quote;
   }
 
-  deleteQuote(): void {
-
+  deleteQuote(quoteId: number): void {
+    this.quoteDeleteService.deleteQuote(quoteId).subscribe();
+    this.selectedQuote = null;
```

Comment:
These 2 lines happen too soon, the Observable needs to complete before reloading really